arroyo-rpc/src/lib.rs

```diff
@@ -190,3 +190,22 @@ pub fn error_chain(e: anyhow::Error) -> String {
         .collect::<Vec<_>>()
         .join(": ")
 }
+
+fn default_async_timeout_seconds() -> u64 {
+    10
+}
+
+fn default_async_max_concurrency() -> u64 {
+    10
```

Comment:
This seems very low to me. If an async udf takes just 50ms to run (pretty typical for a DB lookup) this limits us to 200 events/sec. I'd make this at least 100.